.github/workflows/snapshot-timestamp.yml

```diff
@@ -83,6 +83,16 @@ jobs:
           path: repository/repository/
           retention-days: 5
 
+  if-failed:
+    runs-on: ubuntu-latest
+    needs: [snapshot_and_timestamp]
+    if: needs.snapshot_and_timestamp.result == 'failure'
```

Comment:
Does the token need any other permissions to create issues?

Reply:
The workflow has GITHUB_TOKEN. that has perms

Reply:
looking at it again the workflow only has the 
```
    permissions:
      id-token: 'write'
```

we need to add the issues permission as well

Reply:
Ah, I guess that's per-job permissions. The snapshot-timestamp job has those, i'll make it explicit that the job permission for the failures have issue writes. thank you!